src/netaddress.cpp

```diff
@@ -200,7 +200,21 @@ bool CNetAddr::IsRFC7343() const
            GetByte(13) == 0x00 && (GetByte(12) & 0xF0) == 0x20;
 }
 
-bool CNetAddr::IsTor() const { return m_net == NET_ONION; }
+bool CNetAddr::IsHeNet() const
+{
+    return (GetByte(15) == 0x20 && GetByte(14) == 0x01 && GetByte(13) == 0x04 && GetByte(12) == 0x70);
+}
+
+/**
+ * @returns Whether or not this is a dummy address that maps an onion address
+ *          into IPv6.
+ *
+ * @see CNetAddr::SetSpecial(const std::string &)
+ */
+bool CNetAddr::IsTor() const
+{
+    return (memcmp(ip, pchOnionCat, sizeof(pchOnionCat)) == 0);
+}
```

Comment:
These changes are from somewhere different

Reply:
Must me during merge resolution, will look into it

Reply:
Dash's `IsTor()` comes from https://github.com/dashpay/dash/commit/888f36abe7eded6bd78f614fa2b52f5bbfe1c066, which was done way after these commits. Makes sense why there's a conflict.